addons/resource.language.en_gb/resources/strings.po

```diff
@@ -10976,7 +10976,7 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#20369"
-msgid "Show plot for unwatched items"
+msgid "Show plot and episode thumbnail for unwatched items"
```

Comment:
One option should be enough, I agree (Enabled does A / Disabled does B) its just the exact wording that's neither too long and is fairly clear which is cumbersome to achieve. Which may actually prevent it from happening in one option.

I will wait and see what is suggested that's not too long (not possible with reason) and does this job. Either two options or the length requirement has to be slightly flexible while not going overboard.

I cant add another option, My skills there are limited so Im open to all suggestions and help.